src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
```

Comment:
think about other way to splace your flex-items - your solution differs from the design